lce-rxjava3/src/main/java/com/laimiux/lce/rxjava3/Init.kt

```diff
@@ -11,26 +11,26 @@ import io.reactivex.rxjava3.core.Maybe
 import io.reactivex.rxjava3.core.Observable
 import io.reactivex.rxjava3.core.Single
 
-inline fun <C, E> Completable.toUCE(
+inline fun <C: Any, E> Completable.toUCE(
```

Comment:
Why does `C` have to be `Any`?

Reply:
Type changes within Kotlin/RxJava interop

Reply:
I don't think `Completable` needs it?

Reply:
Since we call `toSingleDefault` on the completable, it is needed.